spec/round_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "./lib/card"
+require "./lib/turn"
+require "./lib/deck"
+require "./lib/round"
+
+
+RSpec.describe(Round) do
+  it("exsits") do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+    deck = [card_1, card_2, card_3]
+    round = Round.new(deck)
+    expect(round).to((be_instance_of(Round)))
+  end
+
+  it("has turns") do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+    deck = [card_1, card_2, card_3]
+    round = Round.new(deck)
+    expect(round.turns).to(eq(turns))
```

Comment:
you don't have a variable or method called `turns` so i imagine this line would error out. If no turns have been taken, we could write the assertion like this: 

`expect(round.turns).to eq([])`